pydash/pydash_web/api_routes.py

```diff
@@ -15,11 +15,43 @@ def login():
     return controller.login()
 
 
+# No @login_required, because this is handled internally.
 @api.route("/api/logout", methods=["POST"])
 def logout():
     return controller.logout()
 
 
+@api.route("/api/user/register", methods=["POST"])
+def register_user():
+    return controller.register_user()
+
+
+# @api.route("/api/user/verify", methods=["POST"])
```

Comment:
If you want to do documentation for a function, use triple-quotes
Example:
def f():
    """This function is responsible for ...."""
    pass

Reply:
This will be removed in the next version, since the API will change to use a POST parameter instead of the URL :+1: